Makefile

```
@@ -0,0 +1,77 @@
+dev_build_version=$(shell git describe --tags --always --dirty)
+
+# TODO: run golint and errcheck, but only to catch *new* violations and
+# decide whether to change code or not (e.g. we need to be able to whitelist
+# violations already in the code). They can be useful to catch errors, but
+# they are just too noisy to be a requirement for a CI -- we don't even *want*
+# to fix some of the things they consider to be violations.
+.PHONY: ci
+ci: deps checkgofmt vet staticcheck ineffassign predeclared test
+
+.PHONY: deps
+deps:
+	go get -d -v -t ./...
+
+.PHONY: updatedeps
+updatedeps:
+	go get -d -v -t -u -f ./...
+
+.PHONY: install
+install:
+	go install -ldflags '-X "main.version=dev build $(dev_build_version)"' ./...
+
+.PHONY: release
+release:
+	@GO111MODULE=on go install github.com/goreleaser/goreleaser
+	goreleaser --rm-dist
+
+.PHONY: docker
+docker:
+	@echo $(dev_build_version) > VERSION
+	docker build --target=gcsemulator -t fullstorydev/gcsemulator:$(dev_build_version) .
+	docker build --target=cbtemulator -t fullstorydev/cbtemulator:$(dev_build_version) .
```

```suggestion
	docker build --target=gcsemulator -t fullstorydev/gcsemulator:$(dev_build_version) -t fullstorydev/gcsemulator:latest .
	docker build --target=cbtemulator -t fullstorydev/cbtemulator:$(dev_build_version) -t fullstorydev/cbtemulator:latest .
```

Otherwise the container test doesn't work.

---

I initially thought that latest tag would be automatically set to the last version we push, but it seems not to be the case by reading this https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375

I think we could reuse the releasing script from grpccurl https://github.com/fullstorydev/grpcurl/blob/master/releasing/do-release.sh